src/unix/linux-syscalls.c

```diff
@@ -26,6 +26,13 @@
 #include <sys/types.h>
 #include <errno.h>
 
+#if defined(__has_feature)
+# if __has_feature(memory_sanitizer)
+# define MSAN_ACTIVE 1
+extern void __msan_unpoison(const volatile void *a, size_t size);
```

Comment:
Is there a reason to declare the prototype manually instead of including msan_interface.h?